src/drawing/DrawingList.tsx

```diff
@@ -34,6 +40,18 @@ export function DrawingList({ activeId = null, annotations, className, onSelect
     const [rootEl, setRootEl] = React.useState<DrawingSVGRef | null>(null);
     const isListening = useIsListInteractive();
 
+    React.useEffect(() => {
+        // Safari doesn't repaint the SVG after a new annotation is drawn which
```

Comment:
This kind of manual manipulation of the render loop doesn't seem ideal. Why doesn't Safari repaint in this scenario?

Reply:
I believe it's due to the fact that Safari/WebKit has some trouble mapping the coordinate system between the SVG and the HTML element the effect is applied to. Applying a property that forces the affected element to render in its own compositor layer seems to make Safari snap back into getting the coordinates right. 

Maybe my wording could be better in the comment above. It's less so the issue of Safari not repainting and more so that because Safari is having trouble mapping the coordinate system which is what is causing the blur to not show up until a repaint occurs.

I could change it to:
`
// Safari has trouble mapping the coordinate system for filters on SVG elements so the fix is to perform a repaint on the SVG to ensure that the coordinate system is mapped correctly.
`

Reply:
If the SVG needs to be in its own composite layer to paint properly, can we add something like `transform: translateZ(0);` to its base class, instead? There are also still other methods to manage layers that don't involve JavaScript, if that doesn't work.